src/nl/rubensten/texifyidea/util/Psi.kt

```diff
@@ -398,3 +398,32 @@ fun PsiFile.hasBibliography() = this.commandsInFileSet().any { it.name == "\\bib
  * @return `true` when the fileset has a bibliography included, `false` otherwise.
  */
 fun PsiFile.usesBiber() = this.commandsInFileSet().any { it.name == "\\printbibliography"}
+
+private fun splitContent(element: PsiElement, delimiter: String = ",") = element.firstChildOfType(LatexNormalText::class)?.text?.split(delimiter)
```

Comment:
I know private functions do not require public documentation, but I think it'd be a wise thing to do here.